.flake8

```diff
@@ -1,2 +1,8 @@
 [flake8]
 max-line-length = 120
+exclude =
```

Comment:
If this is likely to be useful across our repos, could it be added to the flake8 template in the ways of working repo?